pypif_sdk/interop/tests/test_mdf_interop.py

```diff
@@ -0,0 +1,56 @@
+from pypif.obj.common import Property, Scalar, Person, Name, License, Reference, Value
+from pypif.obj.system import System, ChemicalSystem
+from pypif_sdk.interop.mdf import _to_user_defined, _construct_new_key
+
+
+test_pif = ChemicalSystem(
+    chemical_formula="CH4",
+    names = ["methane", "fart"],
+    contacts = [Person(name=Name(given="Albert", family="Einstein")), Person(email="admin@citrine.io")],
+    references = [Reference(doi="doi", authors=[Name(given="Captain", family="Marvel")])],
```

Comment:
What happens to references? The tests seem to only flex properties and conditions.